examples/maven-example/run.bat

```diff
@@ -1 +1 @@
-mvnw spring-boot:run
\ No newline at end of file
+mvn spring-boot:run
```

Comment:
- If you really want to remove the wrapper from the examples, address all of the work outlined in the README.md
- If you are okay with leaving the wrapper in with the example, accept the below suggestion.
```suggestion
mvnw spring-boot:run
```